arch/riscv/Kconfig

```diff
@@ -243,6 +243,7 @@ config PRIVILEGED_STACK_SIZE
 
 config TEST_EXTRA_STACK_SIZE
 	default 4096 if 64BIT
+	default 1024
```

Comment:
Please squash it with the commit which removes this since this should not have been removed in the first place.

Reply:
@dcpleung, again, thanks. my bad, I will squash this